app/Http/Controllers/CelebrityController.php

```diff
@@ -2,9 +2,64 @@
 
 namespace App\Http\Controllers;
 
+use Carbon\Carbon;
+use App\Models\Celebrity;
 use Illuminate\Http\Request;
 
 class CelebrityController extends Controller
 {
-    //
+    public function index() //print all data in Celebrity database
+    {
+        return view('celebrities.index', [
+            'celebrities' => Celebrity::all()
```

Comment:
For this one Libro, I think we need to be careful when getting all the data like this. This can be really heavy on the DB side and not really user-friendly. 

For example:
What would happen if we had 1 million celebrities? 

We should consider:
Should we show all of the results? 
Should we paginate? 
And what about the order of the records?